### ISSUE TITLE
Deeplink not naviagting to specific page

### ISSUE THREAD

**Issue by vignesh-balakrishnan** [ionic-team/ionic-plugin-deeplinks#219]
In ionic 4 when match found i tried navigate user to specific page but it's throwing error.
when I debugged it I found the issue is on deeplink.js file. Ionic 4 is does not have push() function for navigation.
Error messsage:
 **"navController.push is not a function at plugins/ionic-plugin-deeplinks/www/deeplink.js:106"**
Any one please help me to solve this issue thanks in advance.

**Comment by eryzerz** [comment]
Of course, because navcontroller doesn't have push(). Try navigateForward() instead
